src/main/flight/pid.c

```diff
@@ -202,6 +202,7 @@ void resetPidProfile(pidProfile_t *pidProfile)
     pidProfile->pid[PID_ROLL].D = 30;
     pidProfile->pid[PID_PITCH].D = 32;
 #endif
+    pidProfile->motor_output_limit = 100;
```

Comment:
Not that it matters much, but can't we just add this to the macro section above? The stuff below needed to be separated because of macro-in-a-macro problems.